examples/table/config.xml

```diff
@@ -1,10 +1,3 @@
 <View>
-  <Table name="menu" value="$menu" />
-  <Header>Choose ingredients</Header>
-  <Choices name="sub" toName="menu">
-    <Choice value="Tuna" />
-    <Choice value="Vegetables" />
-    <Choice value="Chicken" />
-    <Choice value="Spicy" />
-  </Choices>
-</View>
\ No newline at end of file
+  <RankerBoard name="rankerboard"/>
```

Comment:
Please, create another example, dedicated for this new tag. We need example for Table :)